Add Sleepiq services

Jay2645 opened the pull request:
**Description:**

Added services "set_sleep_number" and "set_to_favorite_sleep_number" to the SleepIQ component. This allows for changing the sleep number of a Sleep Number bed via automations.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30722

## Checklist:

- [ x ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ x ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Jay2645 committed:
Add Sleepiq services

Added services "set_sleep_number" and "set_to_favorite_sleep_number" to the SleepIQ component. This allows for changing the sleep number of a Sleep Number bed via automations.

frenck commented on source/_integrations/sleepiq.markdown:
````diff
@@ -37,3 +37,52 @@ password:
+    side: 'right'
+```
+
+{% configuration %}
````
Please do not use integration configuration block for service calls.

Check other parts of the documentation on how those are normally formatted.

Jay2645 replied:
@frenck Addressed; I was looking at the Domino's component and accidentally thought their configuration was a service call.

frenck commented on source/_integrations/sleepiq.markdown:
```diff
@@ -37,3 +37,52 @@ password:
+
+SleepIQ exposes a few services to control your Sleep Number bed via automations:
+
+This changes a bed's sleep number to the specified value. This value should be a multiple of 5 between 5 and 100. Other values will work, but they will be rounded to the nearest multiple of 5 by the Sleep Number API.
```
Add a title for each service

```### Service `sleepiq.set_sleep_number` ```

Jay2645 replied:
@frenck Addressed.

frenck reviewed (changes requested):
Thanks for the PR @Jay2645!

I've reviewed it and left some comments, could you please take a look? Thanks! 👍

Jay2645 committed:
Change formatting for SleepIQ services

SleepIQ service formatting changed to match formatting seen in the services for the Light integration.

frenck reviewed (approved):
✅ Approved. Can be merged as soon as the parent PR gets merged.
